README.md

```diff
@@ -55,6 +55,13 @@ This directive is always required.
 Maximum HTTP body size this module can generate SXG from. Default value is
 `67108864` (64 MiB).
 
+#### sxg\_cert\_path
+
+The nginx will generate and serve CBOR encoded certificate file in this path.
```

Comment:
**nginx** will generate and serve **the CBOR-encoded** certificate file in this path.

Reply:
Path where nginx will generate and serve the CBOR-encoded certificate file.
ps. What kind of path is this? This is not in the example so a user could be confused.

Reply:
maybe `absolute path` may be good.